src/kernels/helpers.node.ts

```diff
@@ -4,6 +4,7 @@
 'use strict';
 
 import * as path from '../platform/vscode-path/path';
```

Comment:
Do we have linters to ensure nodejs module `path` never gets used.
Or would we want to just replace `path` with `vscode-path` in webpack?

I'd think one of the above would be necessary to ensure we don't use path in web files.

Reply:
We do have a linter that ensures no more usage of 'path' from nodejs anywhere except for '.node' files.